deploy/helm/pulumi-operator/Chart.yaml

```diff
@@ -9,8 +9,8 @@ icon: https://www.pulumi.com/logos/brand/avatar-on-white.svg
 
 type: application
 
-version: 0.9.0
-appVersion: 1.14.0
+version: 2.0.0
+appVersion: "v2.0-devel"
```

Comment:
Rationale for changing the version info:
-  `appVersion` is used as the image tag in deployment.yaml and should match our normal version string.  The prerelease target will replace the value with the tag.
- `version` is the chart version, and I would advocate for doing a major bump.